### DISCUSSION TITLE
Problema de CORS ao tentar renderizar o StatusScreenBrick

### DISCUSSION THREAD

**Discussion by lucaslangsch** [mercadopago/sdk-js#209] [Q&A] [answered]
Olá,

Estou usando as credenciais de teste tanto no front quanto no backend. Eu consigo renderizar normalmente o PaymentBrick e realizar a requisição para criar um pagamento. Recebo o id do pagamento normalmente. Mas quando tento renderizar o Status e o qr code com o StatusScreenBrick tenho problema de CORS

Access to XMLHttpRequest at 'https://http2.mlstatic.com/frontend-assets/op-cho-bricks/build/2.0.2/config/MLB.json?public_key=TEST-8e6e515e-f934-4105-8207-4a1a76a30180&locale=pt-BR&js_version=2.0.0&referer=http%3A%2F%2Flocalhost%3A5173' from origin 'http://localhost:5173' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Observação. Já realizei uma venda usando as credenciais de prod.

**Comment by icaldana** [comment]
Olá Lucas!

Poderia fornecer mais detalhes do erro? Um exemplo de request e se está utilizando alguma tecnologia específica, por exemplo Azure ou Next. Obrigado!

**Comment by lucaslangsch** [comment]
Estou usando React, e usando o sdk-react, agora que percebi que estou no fórum de dúvidas errado

**Reply by icaldana** [reply]
Não se preocupe em relação ao fórum, podemos seguir por aqui mesmo. Poderia por favor enviar um exemplo da request? Uma print da tab network

**Comment by lucaslangsch** [comment]
![Captura de tela de 2024-09-12 14-24-07](https://github.com/user-attachments/assets/d714b5a8-1eee-4bb6-9ecd-b6a87f315a7d)
![Captura de tela de 2024-09-12 14-23-53](https://github.com/user-attachments/assets/e29fa207-dc33-40eb-8987-f489fe275794)
![Captura de tela de 2024-09-12 14-23-26](https://github.com/user-attachments/assets/28e0ea0d-adbe-4e52-8f02-b28a4c8efcd5)

**Comment by lucaslangsch** [comment]
![Captura de tela de 2024-09-12 14-25-26](https://github.com/user-attachments/assets/140c28e7-79b1-47cf-a7f9-861bd9c199a1)

**Comment by icaldana** [comment]
Oi @lucaslangsch, acho que identicamos o problema! Estamos trabalhando para resolver.

Muito obrigado por sua paciência e colaboração! ❤️

**Reply by icaldana** [reply, marked as answer]
@lucaslangsch obrigado por sua paciência, poderia testar novamente? Trabalhamos em uma correção que já está disponível
